lib/l10n/app_en.arb

```diff
@@ -785,5 +801,97 @@
                 "type": "String"
             }
         }
-    }
+    },
+    "wcScopeRequestAccounts": "View your wallet accounts and their balance",
+    "wcScopeSendQubic": "Ask you to send Qubic from your wallet accounts",
+    "wcScopeSendAssets": "Ask you to send Qubic Assets from your wallet accounts",
+    "wcScopeSignTransaction": "Ask you to sign transactions with your private seed",
+    "wcScopeSign": "Ask you to sign messages with your private seed",
+    "wcScopeForbiddenTransfer": "Transfer Qubic or Qubic Assets without your consent",
+    "wcUnknownDapp": "Unknown dApp",
+    "wcAppInfoHeader": "This app is able to",
+    "wcAppInfoHeaderPair": "This app will be able to",
+    "wcAppInfoHeaderPairForbidden": "This app will not be able to",
+    "wcErrorUserDisconnected": "User forcefully disconnected",
+    "wcErrorInvalidURL": "Invalid WalletConnect connection URL",
+    "wcErrorDeprecatedURL": "Deprecated WalletConnect connection URL",
+    "wcErrorUnsupportedChains": "Pair request contains unsupported chains",
+    "wcErrorUnsupportedMethods": "Pair request contains unsupported methods",
+    "wcErrorUnsupportedEvents": "Pair request contains unsupported events",
+    "wcErrorUnsupportedAccounts": "Pair request contains unsupported accounts",
+    "wcErrorUnsupportedNamespaces": "Pair request contains unsupported namespaces",
+    "wcErrorUsedURL": "A connection has already been established via this URL",
+    "wcErrorUnsupportedNetwork": "Unsupported Network",
+    "wcErrorUnsupportedNetworkDescription": "This app uses {networkName} network. Currently, Qubic Wallet only supports apps on Qubic Network",
+    "@wcErrorUnsupportedNetworkDescription": {
+        "placeholders": {
+            "networkName": {
+                "type": "String"
+            }
+        }
+    },
+    "wcRevokePermissions": "Revoke permissions",
+    "wcDappsConnectedNone": "No Active Connections",
+    "wcDappsConnectedNoneHint": "Once you connect to any dapps, you will be able to manage them here",
+    "wcConnectionApproved": "WalletConnect connection approved",
+    "wcConnectionRejected": "WalletConnect connection rejected",
+    "wcConnectionProposalTimeout": "WalletConnect session proposal timed out",
+    "wcPointCameraToQR": "Please point the camera to a QR Code containing the WalletConnect connection info",
+    "wcAddWcTitle": "Pair via WalletConnect",
+    "wcAddURL": "Connect to a dapp by pasting WalletConnect connection URL.",
+    "wcNoPermissions": "This app doesn't have any permissions",
+    "wcOpenApp": "Open App",
+    "wcExpiersInDay": "Session expires in 1 day",
+    "wcExpiersInDays": "Session expires in {days} days",
+    "@wcExpiersInDays": {
```

Comment:
For singular and plural a single string can be use. Please modify it. We are already using such a formats in the app

Reply:
Fixed and commited